### PR TITLE
Sign up page

### PR DESCRIPTION
**Pull request by srishtigautam21**
Added signup page
Files to check:
SignupPage.jsx
AuthContext.js
Sidebar.jsx

### CODE REVIEW

File: src/pages/SignUpPage/SignUpPage.jsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                    //   }
+                  } else {
+                    setError("Email should be valid");
+                  }
+                } else {
+                  setError("Password Should have atleast 8 characters");
+                }
+              } else {
+                setError("Fields cannot be empty");
+              }
```
Comment (MaruthiPothuganti) [inline src/pages/SignUpPage/SignUpPage.jsx:120]: This can be made as a function..😅

**Review by MaruthiPothuganti — APPROVED** [review]
Code looks clean. PR can be merged.

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

File: src/pages/SignUpPage/SignUpPage.jsx (hunk 1 above)
Comment (srishtigautam21) [reply src/pages/SignUpPage/SignUpPage.jsx:120]: yaa right

**Pull request merged**